backend/src/lib/helpers/parseSecretPathSearch.ts

```diff
@@ -0,0 +1,30 @@
+import { removeTrailingSlash } from "@app/lib/fn";
+
+export const parseSecretPathSearch = (search?: string) => {
```

Comment:
If this is used just in `dashboard-rouer` i don't think we need to put this in lib folder

Reply:
moved to `dashboard-router`